dataprofiler/profilers/profile_builder.py

```diff
@@ -1490,6 +1498,7 @@ def _update_correlation(self, clean_samples, prev_dependent_properties):
             batch_corr, batch_properties["mean"],
             batch_properties["std"], batch_samples)
 
+    @utils.method_timeit(name='correlation')
```

Comment:
should we make the name differently as we had correlation above already?
Also, how we name other stats timing to distinguish between update and merge?

Reply:
we don't separate the cost between merge / update of correlations. All together, they are the cost of computing the statistic.